libafl/src/inputs/mod.rs

```diff
@@ -127,12 +130,39 @@ pub trait HasTargetBytes {
     fn target_bytes(&self) -> OwnedSlice<u8>;
 }
 
-/// Contains an internal bytes Vector
-pub trait HasBytesVec {
-    /// The internal bytes map
+/// Contains mutateable and resizable bytes
+pub trait HasMutatorBytes: HasLen {
```

Comment:
We could also split this trait into two; one that only supports static slices, and one that supports growing and shrinking? But it gets (a bit) more messy